mediapipe/__init__.py

```
@@ -11,3 +11,55 @@
+import mediapipe.tasks.python as tasks
+
+
+del framework
+del gpu
+del modules
+del python
+del mediapipe
+del util
+__version__ = '0.1.0.dev0'
+
+from mediapipe.python import *
```

Ruff (0.11.9)

`from mediapipe.python import *` used; unable to detect undefined names

(F403)